sources/packages/web/src/components/common/ProgramDetails.vue

```diff
@@ -153,6 +163,11 @@ export default defineComponent({
       required: true,
       default: {} as EducationProgramAPIOutDTO,
     },
+    isOfferingEditAllowed: {
```

Comment:
Thoughts on changing this to be `isAddOfferingAllowed` as it's specifically used to control visibility of the **Add offering** button? I know the parent uses the same logic for Add/Edit but this component only deals with Add.